cypress/component/ERACommons/era_commons.spec.jsx

```diff
@@ -101,10 +100,32 @@ describe('ERA Commons Component', function () {
       style={{}}
     />);
     cy.get('#era-commons-id').should('exist');
-    cy.get('[data-cy=era-commons-header').should('exist');
-    cy.get('[data-cy=era-commons-required').should('exist');
-    cy.get('[data-cy=era-commons-authenticate-link').should('not.exist');
-    cy.get('[data-cy=era-commons-id-value').should('exist');
+    cy.get('[data-cy=era-commons-header]').should('exist');
+    cy.get('[data-cy=era-commons-required]').should('exist');
+    cy.get('[data-cy=era-commons-authenticate-link]').should('not.exist');
+    cy.get('[data-cy=era-commons-id-value]').should('exist');
     cy.get('.required-field-error-span').should('not.exist');
   });
-});
\ No newline at end of file
+
+  it('shows an error when ECM fails required', function () {
```

Comment:
This change could break the display of an error message on delete failing, and an incorrect token, since those also use the `nihError` code path. If it's not too much work, it would be worth adding tests for those cases too.